test/src/TapeTest.cpp

```diff
@@ -207,7 +207,7 @@ TEST_F(TapeTest, RecordPlayAllCalls)
 	EXPECT_RGL_SUCCESS(rgl_node_rays_set_ring_ids(&setRingIds, rings.data(), rings.size()));
 
 	rgl_node_t setTimeOffsets = nullptr;
-	std::vector<float> timeOffsets = {0.0f, 1.0f, 2.0f};
+	std::vector<float> timeOffsets = {1.0f, 2.0f};
```

Comment:
I am curious about this change :smiley:

Reply:
In an earlier version of this test, both raytraceWithDistortion node and setTimeOffsets node were not included in the graph. When I attached them, I found that the test fails because the number of time offsets does not match the number of rays. Hence the change - we have two rays, so we want two time offsets.